apps/charterafrica/src/pages/Research.page.js

```diff
@@ -0,0 +1,54 @@
+import PageInfo from "@/charterafrica/components/PageInfo";
```

Comment:
1. This page is under knowledge so it should be in `pages/knowledge` folder.
2. Create `pages/knowledge/[slug].js` which should be reusable for both **Research** and **News** depending on value of slug (see `pages/opportunities/[slug].js`)